tests/tests_basics_files.yml

```diff
@@ -47,7 +51,9 @@
       include_tasks: tasks/check_packages.yml
 
     - name: Grep output to messages line
-      command: /bin/grep '\*.info;mail.none;authpriv.none;cron.none.*{{ __default_system_log }}' '{{ __test_default_files_conf }}'
+      command: "/bin/grep
```

Comment:
with `command` it is better to use `>-` which will generate a single line string with each newline replaced with a single space